multimodal/src/autogluon/multimodal/presets.py

```diff
@@ -386,6 +386,31 @@ def medium_quality_faster_inference_image_text_similarity():
 
 
 @automm_presets.register()
+def best_quality_ner():
+    return {
+        "model.names": ["ner"],
+        "model.ner.checkpoint_name": "google/flan-t5-base",
+        "env.precision": 32,
+    }
+
+
+@automm_presets.register()
+def medium_quality_faster_inference_ner():
+    return {
+        "model.names": ["ner"],
+        "model.ner.checkpoint_name": "google/flan-t5-small",
+        "env.precision": 32,
+    }
+
+
+@automm_presets.register()
+def high_quality_fast_inference_ner():
+    return {
+        "model.names": ["ner"],
+        "model.ner.checkpoint_name": "bert-base-cased",
```

Comment:
What's the performance of architectures like `deberta-v3-small`?

Reply:
In general, deberta-v3 works better than bert-base, but
using ddp_spawn + multiple gpus will cause an error.
using ddp + multiple gpus does not lead to good performance.

Reply:
What type of error? Shall we fix it?

Reply:
cuda error, cannot find some functions from cuda.   It may be an error from pytorch lightning.